source/sync/app-builder/device-sync-permissions-guide.txt

```diff
@@ -213,50 +225,51 @@ strategy. To do so, follow these general steps:
 
 .. include:: /includes/use-flex-sync-templates.rst
 
-Now you need to configure the permissions. Under :guilabel:`Define Permissions`,
-use the template dropdown to select the template called "Users can read and 
-write their own data, admins can read and write all data".
+Now you need to configure the permissions. Navigate to the :guilabel:`Rules`
+page. Click the :guilabel:`Default Roles and Filters` button to edit the default
+roles. Use the template dropdown to select the template called "Users can read
+and write their own data, admins can read and write all data".
 
 This populates the rule expression box with the following:
 
 .. code-block:: json
 
-   {
-     "defaultRoles": [
-       {
-         "name": "admin", 
-         "applyWhen": { 
-            "%%user.custom_data.isGlobalAdmin": true 
-         },
-         "read": true,
-         "write": true
-       },
-       {
-         "name": "user", 
-         "applyWhen": {},
-         "read": { "owner_id": "%%user.id" },
-         "write": { "owner_id": "%%user.id" }
-       }
-     ]  
-   }
+   [
+      {
+        "name": "admin", 
+        "apply_when": { 
+          "%%user.custom_data.isGlobalAdmin": true 
+        },
+        "document_filters": {
+          "read": true,
+          "write": true
+        }
```

Comment:
I think we need top-level `"read": true` and `"write": true` in the admin role here as well, or else no fields will have read/write access

Reply:
(feel free to resolve this)